src/shared/types/ThemeColors.ts

```diff
@@ -15,10 +15,9 @@ export const colors = {
         gray: '#9CADB7',
         offwhite: '#D6D2C4',
         concrete: '#95A5A6',
-        red: '#B91C1C', //   Not sure if this should be here, but it's used for remove course, and add course is ut-green
     },
     theme: {
-        red: '#BF0000',
+        red: '#d10000',
```

Comment:
nit: be consistent with capitalization in this file
```suggestion
        red: '#D10000',
```